samples/plusProfileSearch.html

```diff
@@ -76,14 +83,16 @@
         }
 
         var query = document.getElementById('query').value;
-        gapi.client.setApiKey(apiKey);
         var request = gapi.client.plus.people.search({
           'query': query
         });
         request.execute(writeResponse);
       }
     </script>
-    <script src="https://apis.google.com/js/client.js?onload=load"></script>
+    <script async defer src="https://apis.google.com/js/api.js"
+      onload="this.onload=function(){};handleClientLoad();"

```

Comment:
Hey. Shouldn't `handleClientLoad()` be inside `function()` brackets?

`onload="this.onload=function(){handleClientLoad();};"` ?